tests/sanity/tests/channel.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { test } from '@playwright/test'
+import { generateId, PlatformUser } from './utils'
+import { LoginPage } from './model/login-page'
+import { SelectWorkspacePage } from './model/select-workspace-page'
+import { LeftSideMenuPage } from './model/left-side-menu-page'
+import { ChunterPage } from './model/chunter-page'
+import { ChannelPage } from './model/channel-page'
+
+test.describe('channel tests', () => {
+  test('create new private channel tests', async ({ page }) => {
+    const loginPage = new LoginPage(page)
```

Comment:
Why not open a chunter from beginning why we should have extra step to login and select workspace? I support this is not necessary to perform every time. Please check other tests, some of them open a required page from beginning. Thanks,

Reply:
yep. this isn't necessary.
Will update the test. 
Thank you!

Reply:
updated.
cc @haiodo